llama-index-integrations/instrumentation/llama-index-instrumentation-agentops/README.md

```diff
@@ -0,0 +1,14 @@
+# LlamaIndex Instrumentation Integration: Agentops
+
+To integrate AgentOps into your agent workflow,
+simply import `AgentOpsHandler` and add a handler,
+as demonstrated below. Note that all keyword arguments
+anticipated by AgentOps' `AOClient` can be provided
+to this client using the same keyword arguments in
+`add_handler()`.
+
+```
+from llama_index.instrumentation.agentops import AgentOpsHandler
+
+AgentOpsHandler.add_handler()
```

Comment:
all our other existing callback handlers support this sort of `set_global_handler("agentops", **kwargs)` flow -- it would be nice to use the same with the newer callbacks that use instrumentation 

We can then add it to the docs here under the one-click integrations 
https://docs.llamaindex.ai/en/stable/module_guides/observability/?h=observ